db/post_migrate/20230224225129_backfill_svg_sprites.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class BackfillSvgSprites < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def up
+    ThemeField.svg_sprite_fields.find_each(&:upsert_svg_sprite!)
```

Comment:
Is the backfilling necessary? Won't the application upsert the sprite field during a deploy/build? 

I am raising this because we generally don't recommend calling application code in a migration or post migration.

Reply:
It doesn't recompute them on deploy - it doesn't need to.

See dev topic 28750 for why this is a post migration.